spec/controllers/conferences_controller_spec.rb

```diff
@@ -54,4 +54,58 @@
       end
     end
   end
+
+  describe 'POST create' do
+    context 'with an unauthenticated user' do
+      let(:params) { attributes_for :conference }
+
+      it 'cannot create new conference' do
+        expect {
+          post :create,
+          params: { conference: params }
+        }.not_to change { Conference.count }
+      end
+
+      it 'cannot create new conference' do
```

Comment:
Having two examples named identically and internally only diffing by two words makes reading a bit hard here 🙈 
How about adding some more information to the example description or wrapping it into a "context" that says it's about xhr requests now?

Reply:
I'm thinking if it makes sense at all to keep the tests for the http submission in this file. For what I understand, now all files regarding the creation of conferences outside the scope of teams/edit are in the orga folder, meaning that only organizers should have access to them. Am I correct, @klappradla?

Reply:
You're right 💪  makes sense!

I could have seen it earlier, but this constant switching between programming languages and projects today make my brain kinda fuzzy 👻